Yosemite/Yosemite/Model/Storage/BlazeCampaign+ReadOnlyConvertible.swift

```diff
@@ -9,12 +9,12 @@ extension Storage.BlazeCampaign: ReadOnlyConvertible {
     public func update(with campaign: Yosemite.BlazeCampaign) {
         siteID = campaign.siteID
         campaignID = campaign.campaignID
-        productID = nil // TODO: add a new attribute `productURL` and remove `productID`
+        productID = campaign.productID != nil ? NSNumber(value: campaign.productID!) : nil
```

Comment:
Consider replacing force unwrap with an if else condition. 

Also, I am unsure why `productID` is `NSNumber` when it is defined as `Int64` in the model.
<img width="1325" alt="Screenshot 2024-01-12 at 1 53 12 PM" src="https://github.com/woocommerce/woocommerce-ios/assets/524475/c8646557-f9c3-4a3a-9b71-103e008153ae">

Reply:
Updated in f9f956fd97eedc83e33c0dc2ba6027d34520841f.